### PR TITLE
Vim: explicitely check for job support

### LINKED ISSUE #380: Unknown function: job_start
Hi,

it would be good to disable `vim-gitgutter` for vim binaries that are compiled without `job`.

### PR DESCRIPTION
**Pull request by przepompownia**
Closes #380

### CODE REVIEW

File: autoload/gitgutter/async.vim [hunk 1]
```diff
@@ -10,9 +10,11 @@ let s:jobs = {}
 " MacVim-GUI didn't support async until 7.4.1832 (actually commit
 " 88f4fe0 but 7.4.1832 was the first subsequent patch release).
 let s:available = has('nvim') || (
-      \ (has('patch-7-4-1826') && !has('gui_running')) ||
-      \ (has('patch-7-4-1850') &&  has('gui_running')) ||
-      \ (has('patch-7-4-1832') &&  has('gui_macvim'))
+      \ (has('job')) && (
```
Comment (airblade) [inline autoload/gitgutter/async.vim]: I think you can remove the brackets around `has('job')`.

Comment (przepompownia) [inline autoload/gitgutter/async.vim]: Naturally ;)

**Comment by airblade** [comment]
Thanks!  It's in 64a445b.

**Comment by przepompownia** [comment]
Thank you for hint (direct link in #380).

**Pull request closed without merging**